saleor/app/models.py

```diff
@@ -117,6 +117,10 @@ def create(self, app, name="", auth_token=None, **extra_fields):
         app_token.save()
         return app_token, auth_token
 
+    def create_with_token(self, *args, **kwargs):
+        """See https://app.clickup.com/t/2549495/SALEOR-7847 ."""
```

Comment:
This is more of a comment than a docstring. Also, the URL is not public, so most people won't be able to understand what the docstring is trying to convey.